next.config.js

```diff
@@ -22,6 +22,7 @@ const SCRIPT_SRC = [
   'https://static.hotjar.com',
   'https://script.hotjar.com',
   'https://*.juicebox.money',
+  'https://qwestive-referral-prod.web.app',
```

Comment:
**question**: Is it possible that we can instead bundle this code directly into our app? It would be nice to avoid the need to add this to our CSP.

Reply:
replied in discord chat. 
As discussed, I will be adding the 2 scripts in public directory.

Reply:
Hey @tomquirk 

Hosting scripts on JBM, didn't worked coz we strictly check origin within iframe logic which only accepts incoming events from our scripts. 

Let's go with setup as of now, and later we can update this npm package as discussed and we would avoiding adding anything to CSP.

Reply:
Hey @tomquirk

Hosting scripts on JBM, didn't worked coz we strictly check origin within iframe logic which only accepts incoming events from our scripts.

can we go ahead with setup as of now, and later we can update this with npm package as discussed and we would avoiding adding anything to CSP.